src/D2L.CodeStyle.TestAnalyzers/Common/Diagnostics.cs

```diff
@@ -2,25 +2,6 @@
 
 namespace D2L.CodeStyle.TestAnalyzers.Common {
 	public static class Diagnostics {
-		public static readonly DiagnosticDescriptor TestCaseSourceStrings = new DiagnosticDescriptor(
-			id: "D2LTESTS001",
-			title: "Use nameof in TestCaseSource attributes.",

```

Comment:
https://github.com/nunit/nunit.analyzers/blob/master/documentation/NUnit1002.md

Reply:
Warning vs Error matter for us?

Reply:
We set `TreatWarningsAsErrors=true` globally, but we could up these ones to errors in the `.editorconfig` if we wanted to be more explicit.